subsys/bluetooth/services/hids.c

```diff
@@ -392,6 +392,12 @@ static ssize_t hids_feat_rep_write(struct bt_conn *conn,
 				   void const *buf, u16_t len, u16_t offset,
 				   u8_t flags)
 {
+	/* Write command operation is not allowed for this characteristic. */
+	if ((flags & BT_GATT_WRITE_FLAG_CMD) == BT_GATT_WRITE_FLAG_CMD) {
```

Comment:
We can simplify:
`if (flags & BT_GATT_WRITE_FLAG_CMD)`

Reply:
There is also one more flag which can be set, in this case if I simplify it, this condition will be still true for other flag.

Reply:
Are you sure this will apply for `if (flags & BT_GATT_WRITE_FLAG_CMD)` as a condition?
This should mask only the `BT_GATT_WRITE_FLAG_CMD` flag and work in the same way as your condition (its just shorter).

Reply:
Ah. I forgot that this is just a bit value. It can be simplify :)

Reply:
Done